modules/nw-configuring-ingress-cluster-traffic-aws-networking-load-balancer.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: CONCEPT
+[id="nw-configuring-ingress-cluster-traffic-aws-network-load-balancer_{context}"]
+= Configuring ingress cluster traffic on AWS using a Network Load Balancer
+
+{product-title} provides methods for communicating from outside the cluster with services running in the cluster. This method uses a Network Load Balancer (NLB). You can configure an NLB on a new or existing AWS cluster.
```

Comment:
```suggestion
{product-title} provides methods for communicating from outside the cluster with services that run in the cluster. This method uses a Network Load Balancer (NLB). You can configure an NLB on a new or existing AWS cluster.
```

I know what you mean here, but I find "This method uses..." a little confusing to parse immediately following the first sentence, which introduces multiple methods. 

The idea is "one such method uses...", right?

Reply:
Yeah I see what you mean...I've updated the text to be more clear about this